lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -96,11 +96,13 @@ class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
 		'h6'      => 'h6',
 		'button'  => '.wp-element-button, .wp-block-button__link', // We have the .wp-block-button__link class so that this will target older buttons that have been serialized.
 		'caption' => '.wp-element-caption, .wp-block-audio figcaption, .wp-block-embed figcaption, .wp-block-gallery figcaption, .wp-block-image figcaption, .wp-block-table figcaption, .wp-block-video figcaption', // The block classes are necessary to target older content that won't use the new class names.
+		'cite'    => '.wp-element-cite, .wp-block-quote cite, .wp-block-quote__citation, .wp-block-pullquote cite, .wp-block-pullquote__citation',
```

Comment:
Are there cases when we _don't_ want to style the `cite` element? I'm just wondering if we can simplify this to be just `cite`?

Reply:
It started out as only cite 😄 but then I read through the previous 'elements' pull requests and I thought the class names were added to make it easier for developers.

Reply:
The class names are necessary if we expect there to be situations where we do NOT want elements to pick up this style - Buttons is a good example - there might be button elements which we don't want to style as buttons, and link elements which we do want to appear as buttons. Links on the other hand should always use the link style rules, so we use a much simpler selector. If we can use a `cite` selector here it will make things much simpler. The only reason I can think of not doing that is if there are blocks that might use `cite` but don't want these styles, but I can't think of a case when that would be true.